src/utils/__test__/SolutionsUtils.spec.js

```diff
@@ -94,4 +94,50 @@ describe('addRunTemplatesParametersIdsDict for a minimal or incomplete solution'
       runTemplatesParametersIdsDict: {},
     });
   });
+  test('if parameters constraints are valid', () => {
+    const spyConsoleWarn = jest.spyOn(console, 'warn').mockImplementation();
+    const solution = {
+      parameters: [
+        {
+          id: 'parameter1',
+          varType: 'int',
+          options: {
+            validation: '> parameter2',
+          },
+        },
+        {
+          id: 'parameter2',
+          varType: 'int',
+          options: null,
+        },
+      ],
+    };
+    SolutionsUtils.checkParametersValidationConstraintsInSolution(solution);
+    expect(spyConsoleWarn).toHaveBeenCalledTimes(0);
+  });
+  test('if parameters constraints are not valid', () => {
+    const spyConsoleWarn = jest.spyOn(console, 'warn').mockImplementation();
+    const warning =
+      'Constraint "> parameter2" cannot be applied to parameter \n        with id "parameter1", ' +
+      'please check your solution configuration file';
+    const solution = {
+      parameters: [
+        {
+          id: 'parameter1',
+          varType: 'int',
+          options: {
+            validation: '> parameter2',
+          },
+        },
+        {
+          id: 'parameter2',
+          varType: 'string',
+          options: null,
+        },
+      ],
+    };
+    SolutionsUtils.checkParametersValidationConstraintsInSolution(solution);
+    expect(spyConsoleWarn).toHaveBeenCalledTimes(1);
+    expect(spyConsoleWarn).toHaveBeenCalledWith(warning);
```

Comment:
Great idea, it wasn't easy to hard code **exactly** the same message and I was afraid it would be brittle, thanks for this more robust suggestion :slightly_smiling_face: